.github/workflows/nightlies.yaml

```
@@ -29,34 +29,30 @@ jobs:
         with:
           python-version: '3.10'
       - name: Set up Node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 16
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: '2.7'
       - name: Set up JDK ${{ matrix.java }}
```

actionlint (1.7.4)

property "java" is not defined in object type {}

(expression)